relate/bin/relate.py

```diff
@@ -305,6 +343,14 @@ def main() -> None:
     parser_expand_yaml.add_argument("YAML_FILE")
     parser_expand_yaml.set_defaults(func=expand_yaml_ui)
 
+    parser_lint_yaml = subp.add_parser("lint_yaml")
+    parser_lint_yaml.add_argument("--repo-root", default=os.getcwd())
+    parser_lint_yaml.add_argument("--config-file", default="./.yamllint")
+    parser_lint_yaml.add_argument("--directories", metavar="DIRECTORIES",
+                                  nargs="*")
+    parser_lint_yaml.add_argument("--files", metavar="FILES", nargs="*")
```

Comment:
No need to force the user to keep them separate: `os.path.isdir`. :slightly_smiling_face: